docs/xbcloud-binary-fifo-datasink.md

```diff
@@ -0,0 +1,64 @@
+# FIFO data sink
+
+The feature is in [tech preview](glossary.md#tech-preview).
+
+[Percona XtraBackup 8.0.33-28](release-notes/8.0/8.0.33-28.0.md) implements a new data sink that uses the first in, first out (FIF0) method. With the `FIFO` data sink feature, you can benefit from faster backups by streaming data in parallel from XtraBackup to `xbcloud`.
+
```

Comment:
Lets move the usage/examples section to the top

Reply:
I remember from my discussion with marce that --fifo-streams value has to match on both sides. 
ie on backup sides if we use 3 and xbcloud side if we use 2, it will fail.

Reply:
Updated